README.md

```diff
@@ -3,131 +3,86 @@
 
-This library makes possible to receive a JSON encoded request to a gRPC service. It provides an implementation-agnostic module for mapping to
-your favorite HTTP server as well as few implementations for direct usage in some well-known HTTP servers.  
-For requests/responses mapping a [standard GPB <-> JSON mapping](https://developers.google.com/protocol-buffers/docs/proto3#json) is used.
+This library allows to make a JSON encoded request to a gRPC service. It provides an implementation-agnostic module for mapping to your favorite HTTP server (`core`) as well as few implementations for direct usage in some well-known HTTP servers.  
 
-It uses Scala macros for creating mapping between runtime-provided service and method names to pregenerated Java gRPC classes. In case you
-don't want to use _plain Java API_ you can easily use it together with [Cactus](https://github.com/avast/cactus).
+[Standard GPB <-> JSON mapping](https://developers.google.com/protocol-buffers/docs/proto3#json) is used.
 
-The API is _finally tagless_ (read more e.g. [here](https://www.beyondthelines.net/programming/introduction-to-tagless-final/))
-meaning it can use whatever [`F[_]: cats.effect.Effect`](https://typelevel.org/cats-effect/typeclasses/effect.html)
-(e.g. `cats.effect.IO`, `monix.eval.Task`).
+The API is _finally tagless_ (read more e.g. [here](https://www.beyondthelines.net/programming/introduction-to-tagless-final/)) meaning it can use whatever [`F[_]: cats.effect.Effect`](https://typelevel.org/cats-effect/typeclasses/effect.html) (e.g. `cats.effect.IO`, `monix.eval.Task`).
 
-There are several modules:
-1. core - for basic implementation-agnostic usage
-1. [http4s](http4s) - integration with [http4s](https://http4s.org/) webserver
-1. [akka-http](akka-http) - integration with [Akka Http](https://doc.akka.io/docs/akka-http/current/server-side/index.html) webserver
-
-The created [`GrpcJsonBridge`](core/src/main/scala/com/avast/grpc/jsonbridge/GrpcJsonBridge.scala) exposes not only the methods itself but
-also provides their list to make possible to implement an _info_ endpoint (which is already implemented in server-agnostic implementations).
-
-Recommended URL pattern for exposing the service (and the one used in provided implementations) is `/$SERVICENAME/$METHOD` name and the http
-method is obviously `POST`. The _info_ endpoint is supposed to be exposed on the `/$SERVICENAME` URL and available with `GET` request.
-
-The bridge passes all received headers from JSON request to the gRPC service so it's not any problem to use it e.g. authentication.
-
-## Core module
-
-### Dependency
-
-#### Gradle
+## Usage
 ```groovy
 compile 'com.avast.grpc:grpc-json-bridge-core_2.12:x.x.x'
 ```
-
-#### SBT
 ```scala
 libraryDependencies += "com.avast.grpc" %% "grpc-json-bridge-core" % "x.x.x"
 ```
 
-### Usage
```

Comment:
I don't really like what you did with the readme. What's the problem with current usage examples?

Reply:
The problem is that usage is completely different now :) I also tried to use it more concise, and don't split it to multiple files.

Reply:
Proto definition is now in the README.